### ISSUE TITLE
<Error occurred, plz report to Optional[Jade]>

### ISSUE THREAD

**Issue by andrijzyn** [Snownee/Jade#414]
### Mod loader

Forge

### Minecraft version

1.20.1

### Mod version

11.9.3

### Modloader version

47.3.0

### Modpack info

https://drive.google.com/file/d/1No2xI347-q6XROa_IqyeljsTNN9imqTI/view?usp=sharing

### [IMPORTANT] If bug:

- [x] I have confirmed this bug is reproducible on a minimal setup, not a modpack with many other mods.

### [IMPORTANT] If bug: The **latest.log** file, not the crash report

https://pastebin.com/hb1Xppyr

### Issue description
This is custom TerraFirmaGreg modpack*
![зображення](https://github.com/Snownee/Jade/assets/62100293/9c83e425-d221-475c-a2fd-8176b8ccac5e)

**Comment by truepurpledino** [comment]
Having this as well on [Craftopia](https://legacy.curseforge.com/minecraft/modpacks/craftoria/files/5441979) (version 15.0.4+neoforge of Jade), whilst looking at a seemingly random assortment of blocks.

**Comment by dkzerahh** [comment]
This is also happening with some vanilla blocks from Minecraft 1.21 (I saw it with Vine and Mangroove Leaves)

NeoForge 21.0.17
[JadeErrorOutput.txt](https://github.com/user-attachments/files/15893882/JadeErrorOutput.txt)

**Comment by Slexom** [comment]
Same issue with Craftoria modpack, MC 1.21, Neoforge 21.0.16.
Seems to be related to any shearable block like leaves, vines, grass and so on.
Here the log

_JadeErrorOutput.txt_
```txt
Jun 19, 2024, 12:50:57 AM
snownee.jade.addon.harvest.HarvestToolProvider@2236b532
java.lang.NoSuchFieldError: Class net.neoforged.neoforge.common.Tags$Items does not have member field 'net.minecraft.tags.TagKey TOOLS_SHEARS'
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.util.CommonProxy.isShears(CommonProxy.java:208)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.addon.harvest.HarvestToolProvider.getText(HarvestToolProvider.java:160)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.addon.harvest.HarvestToolProvider.appendTooltip(HarvestToolProvider.java:121)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.addon.harvest.HarvestToolProvider.appendTooltip(HarvestToolProvider.java:41)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.impl.BlockAccessorClientHandler.gatherComponents(BlockAccessorClientHandler.java:93)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.impl.BlockAccessorClientHandler.gatherComponents(BlockAccessorClientHandler.java:24)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.overlay.WailaTickHandler.tickClient(WailaTickHandler.java:194)
	at TRANSFORMER/jade@15.0.4+neoforge/snownee.jade.util.ClientProxy.onClientTick(ClientProxy.java:176)
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.ConsumerEventHandler.invoke(ConsumerEventHandler.java:26)
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:350)
	at MC-BOOTSTRAP/net.neoforged.bus/net.neoforged.bus.EventBus.post(EventBus.java:315)
	at TRANSFORMER/neoforge@21.0.16-beta/net.neoforged.neoforge.client.ClientHooks.fireClientTickPost(ClientHooks.java:1052)
	at TRANSFORMER/minecraft@1.21/net.minecraft.client.Minecraft.tick(Minecraft.java:1918)
	at TRANSFORMER/minecraft@1.21/net.minecraft.client.Minecraft.runTick(Minecraft.java:1164)
	at TRANSFORMER/minecraft@1.21/net.minecraft.client.Minecraft.run(Minecraft.java:810)
	at TRANSFORMER/minecraft@1.21/net.minecraft.client.main.Main.main(Main.java:230)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at MC-BOOTSTRAP/fml_loader@4.0.6/net.neoforged.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:136)
	at MC-BOOTSTRAP/fml_loader@4.0.6/net.neoforged.fml.loading.targets.CommonLaunchHandler.clientService(CommonLaunchHandler.java:124)
	at MC-BOOTSTRAP/fml_loader@4.0.6/net.neoforged.fml.loading.targets.CommonClientLaunchHandler.runService(CommonClientLaunchHandler.java:32)
	at MC-BOOTSTRAP/fml_loader@4.0.6/net.neoforged.fml.loading.targets.CommonLaunchHandler.lambda$launchService$4(CommonLaunchHandler.java:118)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:103)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:74)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@11.0.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.run(BootstrapLauncher.java:210)
	at cpw.mods.bootstraplauncher@2.0.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:69)

```

**Comment by Slexom** [comment]
15.0.5 solved for neoforge on MC 1.21
